### DISCUSSION TITLE
Avatar update - Not working on Vercel

### DISCUSSION THREAD

**Discussion by sudipkundu999** [vercel/community#5198] [Bug Report]
### Page to Investigate

https://vercel.com/account

### Steps to Reproduce

Hi, I tried to change my avatar, but after changing I'm unable to see any avatar. Screenshot attached.

<img width="1440" alt="image" src="https://github.com/vercel/community/assets/95222845/a418e01e-b9c2-4f42-9bd1-6b68ad39bd87">

**Comment by amyegan** [comment]
Thanks for reporting this. I pass this info along to our engineering team and we'll post updates when we know more about the cause

**Reply by sudipkundu999** [reply]
Just checked. Working as expected now.

**Reply by amyegan** [reply]
The team shipped a more permanent fix today. Please let me know if you still have any trouble setting an avatar. 🙏

**Comment by Moatezz** [comment]
same issue with me, but after a few minutes it displayed properly
